hexrd/fitting/peakfunctions.py

```diff
@@ -537,10 +537,10 @@ def _gaussian_pink_beam(p, x):
     g[~zmask] = \
         (0.5*(alpha*beta)/(alpha + beta)) * np.exp(u[~zmask])*t1[~zmask] \
         + np.exp(v[~zmask])*t2[~zmask]
-
     mask = np.isnan(g)
     g[mask] = 0.
-    g *= A
+    gmax = g.max()
+    g *= A/gmax
```

Comment:
```suggestion
    g *= A / g.max()
```

This is shorter, but a not a necessary change.